packages/utilities/src/utils/common.ts

```diff
@@ -320,19 +320,21 @@ export const getContainer = async (
   if (!remoteContainer) {
     throw Error(`Remote container options is empty`);
   }
+  // @ts-ignore
```

Comment:
anyone know TS wanna help fix that ignore? :P

Reply:
Done :heavy_check_mark: